core/tabs/system-setup/compile-setup.sh

```diff
@@ -40,6 +40,13 @@ installDepend() {
         apk)
             "$ESCALATION_TOOL" "$PACKAGER" add build-base multitail tar tree trash-cli unzip cmake jq
             ;;
+        xbps-install)
+            COMPILEDEPS='base-devel'
+            "$ESCALATION_TOOL" "$PACKAGER" -y "$DEPENDENCIES" "$COMPILEDEPS"
```

Comment:
```suggestion
            "$ESCALATION_TOOL" "$PACKAGER" -y $DEPENDENCIES $COMPILEDEPS
```

Reply:
Please see [SC2086](https://www.shellcheck.net/wiki/SC2086).

Reply:
Except I get this "Package 'tar tree multitail tldr trash-cli unzip cmake make jq' not found in repository pool.", with double quotes. Suggestions?

Reply:
oh okay. do this then:
```suggestion
            # shellcheck disable=SC2086
            "$ESCALATION_TOOL" "$PACKAGER" -y $DEPENDENCIES $COMPILEDEPS
```